Alternative for resizable dock (#67)
3liz/lizmap-javascript-scripts: library/ui/resize_dock_2/README.md

# Resize dock 2

> I had some bugs with Resize Dock with splitter](./library/ui/resize_dock_with_splitter) so I made an alternative using [interact.js](https://interactjs.io/).

Reviewer: ```suggestion
An alternative implementation of the other script [resize dock with splitter](./resize_dock_with_splitter), using [interact.js](https://interactjs.io/).
```
